notebook/tests/selenium/test_display_id.py

```diff
@@ -0,0 +1,160 @@
+"""
+Various output tests.
+"""
+import pytest
+import textwrap
+
+from .utils import wait_for_script_to_return_true
+
+INITIAL_CELLS = [textwrap.dedent("""\
+    ip = get_ipython()
+    from IPython.display import display
+    def display_with_id(obj, display_id, update=False, execute_result=False):
+        iopub = ip.kernel.iopub_socket
+        session = get_ipython().kernel.session
```

Comment:
I'm guessing it's just an oversight by the original author that this line uses `get_ipython()` instead of the `ip` variable (like the other lines)?

```suggestion
        session = ip.kernel.session
```